packages/UI/src/components/Button/Button.tsx

```diff
@@ -1,116 +1,55 @@
 import styled, { css } from 'styled-components';
 
 interface ButtonProps {
-  bgColor?: string;
-  textColor?: string;
-  marginTop?: number;
-  width?: number;
-  fontSize?: number;
-  hero?: boolean;
-  modal?: boolean;
-  intro?: boolean;
+  buttonType?: string;
+  buttonSize?: string;
 }
 
-const shadeColor = (color: string, percent: number): string => {
-  let R = parseInt(color.substring(1, 3), 16);
-  let G = parseInt(color.substring(3, 5), 16);
-  let B = parseInt(color.substring(5, 7), 16);
-
-  R = parseInt(((R * (100 + percent)) / 100).toString(), 10);
-  G = parseInt(((G * (100 + percent)) / 100).toString(), 10);
-  B = parseInt(((B * (100 + percent)) / 100).toString(), 10);
-
-  R = R < 255 ? R : 255;
-  G = G < 255 ? G : 255;
-  B = B < 255 ? B : 255;
-
-  const RR =
-    R.toString(16).length === 1 ? `0${R.toString(16)}` : R.toString(16);
-  const GG =
-    G.toString(16).length === 1 ? `0${G.toString(16)}` : G.toString(16);
-  const BB =
-    B.toString(16).length === 1 ? `0${B.toString(16)}` : B.toString(16);
-
-  return `#${RR}${GG}${BB}`;
-};
-
-const colorUnderButtons = (hexColor: string) => {
-  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);
-  return result
-    ? `rgba(${parseInt(result[1], 16) - 102},${parseInt(result[2], 16) - 55},${
-        parseInt(result[3], 16) - 14
-      },1)`
-    : null;
-};
-
-const onHoverBackgroundColor = (hexColor: string) => {
-  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);
-  return result
-    ? `rgb(${parseInt(result[1], 16) + 44},${parseInt(result[2], 16) + 44},${
-        parseInt(result[3], 16) + 44
-      })`
-    : null;
-};
-const Button = styled.a<ButtonProps>`
-  font-family: ${({ theme }) => theme.fonts.headline}, sans-serif;
-  background-color: ${({ theme, bgColor }) => bgColor || theme.colors.ACCENT_1};
-  color: ${({ theme, textColor }) => textColor || theme.colors.NEUTRAL_2};
-  transition: background-color 0.5s, color 0.5s;
-
-  width: ${({ width }) => `${width}em` || ''};
-  border: 0px;
-  border-bottom: 3px solid
-    ${({ theme, bgColor }) =>
-      colorUnderButtons(bgColor || theme.colors.ACCENT_1)};
+const Button = styled.a.attrs(({ disabled }) => ({
+  disabled,
+}))<ButtonProps>`
+  font-family: ${({ theme }) => theme.fonts.normal};
```

Comment:
`font-family, font-size, font-weigh, line-height, text-transform` should all be removed from this file and we can import them from the typography atom that I made like this

```ts
import { typographyStyles } from '../Typography/Styled.Typography';
.
.
.
const Button = styled.button.attrs(({ disabled }) => ({
  disabled: disabled,
}))<ButtonProps>`
   ${typographyStyles['button']}
   ...rest of your styles
`
```